Add AM3 specific run docs

Whyborn opened the pull request:
# ACCESS-Hive Docs

## Description

Adds a bare-minimum page that, in combination with the in-progress [*Rose/Cylc*](https://github.com/ACCESS-NRI/ACCESS-Hive-Docs/pull/1033) docs, describes briefly how to run AM3.

## Type of change

- [x] New link / content

## Checklist:

- [ ] The new content is accessible and located in the appropriate section
- [ ] My changes do not break navigation and do not generate new warnings
- [ ] I have checked that the links are valid and point to the intended content
- [ ] I have checked my code/text and corrected any misspellings

Whyborn committed:
Create quick start for AM3

Whyborn committed:
Create quick start for AM3

Whyborn committed:
Minor change to wording

Whyborn committed:
Minor change to wording

Whyborn commented:
I did think about whether they should be separate or a single PR. I came to the conclusion that they should be separate because the model page in #1056 can be merged independently of the Rose/Cylc docs, while this shouldn't be merged before the Rose/Cylc docs are finalised.

atteggiani commented:
I understand and that's true. 
However, both pages relate to AM3 release, and they should be merged at the same time whenever the Beta (or Alpha) release happens.

I think the Rose/Cylc page will be ready before we're ready for AM3 release (i.d., before we're ready to merge these pages), so I would not see that a a problem.

Whyborn committed:
Add changes from add-AM3-model-page branch

Whyborn committed:
Add changes from add-AM3-model-page branch

Whyborn committed:
Add changes from add-AM3-model-page branch

Whyborn committed:
Add changes from add-AM3-model-page branch

ccarouge commented on docs/models/run-a-model/run-access-am.md:
```diff
@@ -0,0 +1,21 @@
+{% set model = "ACCESS-AM3" %}
+{% set github_configs = "https://github.com/ACCESS-NRI/access-am3-configs" %}
+{% set github_ssh = "git@github.com:ACCESS-NRI/access-am3-configs.git" %}
+{% set configs_docs = "https://access-om3-configs.access-hive.org.au" %}
```
Wrongly pointing to OM3 configs.

ccarouge reviewed (dismissed):
A few comments, nothing major.

Whyborn committed:
Add access to list of projects

Whyborn committed:
Add access to list of projects

Whyborn committed:
Add access to list of projects

Whyborn committed:
Add access to list of projects

ccarouge commented on docs/models/access-am.md:
```diff
@@ -0,0 +1,28 @@
+
+- **Land**: [CABLE](/models/model_components/land#cable).
+
+- **Land Biogeochemistry**: [CASA-CNP](/models/model_components/l`and_bgc#casa_cnp).
```
```suggestion
- **Land Biogeochemistry**: [CASA-CNP](/models/model_components/bgc_land#casa_cnp).
```

Whyborn commented on docs/models/access-am.md:
```diff
@@ -0,0 +1,28 @@
+{% set model = "ACCESS-AM3" %}
+## {{ model }}
+
+{{ model }} is a suite of coupled atmosphere-land configurations developed by ACCESS-NRI, [CSIRO]() and the [ARC Centre of Excellence for Weather in the 21st Century].
```
This reads a bit funny to me- I think it should either be the "the Commonwealth Science and Industrial Research Organisation (CSIRO)" or just "CSIRO" without "the". And I've left the hyperlink blank at the moment, because I wasn't sure whether to link to the most generic CSIRO page or find something more specific to the earth sciences.

atteggiani commented on docs/models/index.md:
```diff
@@ -1,54 +1,127 @@
+            <span class="bold" >ACCESS-AM</span>
             <span>
-                Overview and descriptions of the ACCESS models supported by ACCESS-NRI
+                ACCESS-AM is a global atmospheric model that includes atmosphere, land, atmospheric chemistry, aerosols and land biogeochemsitry components.
```
Yes we should. Thank you for picking it up.

Whyborn committed:
Minor fixes

Whyborn committed:
Minor fixes

Whyborn committed:
Minor fixes

Whyborn committed:
Minor fixes

Whyborn committed:
reverse land_bgc

Whyborn committed:
reverse land_bgc

Whyborn committed:
reverse land_bgc

Whyborn committed:
reverse land_bgc

Whyborn committed:
Fill in missing hyperlinks

Whyborn committed:
Fill in missing hyperlinks

Whyborn committed:
Fill in missing hyperlinks

Whyborn committed:
Fill in missing hyperlinks

Whyborn committed:
Change suite to set

Whyborn committed:
Change suite to set

Whyborn committed:
Change suite to set

Whyborn committed:
Change suite to set

Whyborn committed:
Be slightly more verbose in Quick start section

Whyborn committed:
Be slightly more verbose in Quick start section

Whyborn committed:
Merge commit

Whyborn committed:
Add message about getting access to repos and link to config docs

Whyborn committed:
Add message about getting access to repos and link to config docs

atteggiani committed:
Fixes after rebasing

atteggiani committed:
Added access-am3-configs private repo to link checker exclusions, because it would fail for permission reasons

atteggiani commented:
Hi @Whyborn, thank you for this PR. 
I will review in the upcoming days. 

I see there is a draft PR (#1000) that looks related to the same content. 
Do you know if this PR supersedes that one? Can it be deleted?

Whyborn commented:
Yep, that PR has been superseded. I have closed it.

Whyborn commented on docs/models/run_a_model/run_access-am3.md:
Well we would have similar information for both- simply links to other locations, myNCI to request to join NCI projects, and to the forum to request access to our repos. I can only think of 3 possible avenues for people to request access:

* The forum, either by replying to a thread or directly messaging one of the listed people.
* Github, via an issue on the access-am3-configs-doc repo
* Direct email to one of the repo admins

I don't think the last option is a good one, and the first 2 both have downsides. For the forum, it has the extra step of requiring people to make an account (which we would encourage people to do anyway). For Github, I think it would be easier for a reply on an issue to get lost in the noise, and there are no direct messaging options AFAIK.

Did you have something else in mind, other than a link to either a publicly available Github issue or a forum topic?

Whyborn committed:
Merge branch '1039-AM3-run-a-model-docs' of github.com:ACCESS-NRI/ACCESS-Hive-Docs into 1039-AM3-run-a-model-docs

ccarouge committed:
Reorganise the information

ccarouge commented:
Reading this, I wondered if it would work better if it followed the other Run a Model pages structure more closely. So I've pushed a reorganised page in a separate branch: https://github.com/ACCESS-NRI/ACCESS-Hive-Docs/blob/alternate-run-AM3/docs/models/run_a_model/run_access-am3.md (sorry @atteggiani , you probably don't want rogue branches on ACCESS-Hive-docs).

@Whyborn @atteggiani @heidinett let me know what you think.

Note: I haven't previewed the alternate, so there might be errors, just looking at global page organisation feedback

ccarouge commented:
@Whyborn , the branch I created stems from this branch. I'll just do a fast-forward merge in this branch so this PR can still update both the Model and Run a Model pages.

ccarouge commented:
Done. This branch now has the reorganised Run a Model page.

Whyborn commented:
Oh right, I didn't notice that. Thanks

ccarouge committed:
Added ACCESS-AM model page to navigation

Whyborn committed:
Remove rogue files and add AM3 to models index in mkdocs.yml

Whyborn committed:
merge mkdocs.yml

ccarouge committed:
Correct path to ACCESS-AM logo

Whyborn committed:
Add link to specific section of rose/cylc docs in last paragraph

Whyborn committed:
Add AM3 model card to run a model page

Whyborn committed:
Add correct link to github section

Whyborn committed:
Remove misnamed run_access_am file

Whyborn committed:
Add AM3 mentions to the appropriate model components

Whyborn commented:
The current image we have at `/assets/model-config-logos/model_visualisation/access_am_model_visualisation.png` doesn't look appropriate for AM3- looks like something from an ocean/sea-ice simulation? Something like the ESM visualisation would be more appropriate for AM.

Whyborn committed:
Fix AM run a model card

ccarouge commented on docs/models/access_models/access-am.md:
```diff
@@ -0,0 +1,26 @@
+
+![ACCESS AM model](/assets/model-config-logos/configurations-without-titles/access-am.png){: class="img-contain white-background round-edges with-padding intro-img" loading="lazy"}
+
+The ACCESS Atmosphere Model (ACCESS-AM) is a global coupled atmospheric model that includes [atmosphere](/models/model_components/atmosphere), [aerosols and atmospheric chemistry](/models/model_components/aerosols_atmospheric_chemsitry), [land](/models/model_components/land) and [land biogeochemistry](/models/model_components/bgc_land).
```
There is no atmospheric chemistry for the moment. But atm. chemistry is always an optional add-on (people are working on chemistry for AM3), just like biogeochem. 

We could rewrite this as:


```suggestion
The ACCESS Atmosphere Model (ACCESS-AM) is a global coupled atmospheric model that can include [atmosphere](/models/model_components/atmosphere), [aerosols and atmospheric chemistry](/models/model_components/aerosols_atmospheric_chemsitry), [land](/models/model_components/land) and [land biogeochemistry](/models/model_components/bgc_land), depending on the configuration.
```

Whyborn commented:
> As far as I can tell, these aren't used anywhere on the site.

They're used as the cards on the [Run a Model](https://docs.access-hive.org.au/pr-previews/1058/models/run_a_model/) landing page.

Whyborn committed:
Change wording slightly to make clear all components are necessarily always active

Whyborn commented:
Just doing some thinking about what order to do things when we actually want to make the release. We have a few links which _will be_ pointing to forum posts when we release, and vice versa. I think the order of operations should be:

1. Have this PR pass review, with the links to the forum empty.
2. Publish the Alpha release, feedback and request for access topics to the forum, with links to the PR preview.
3. Update the empty links in the PR to the relevant forum posts and merge the PR.
4. Edit the posts on the forum to point to the main docs pages instead of the PR preview.

@atteggiani does this sound reasonable to you?

Whyborn committed:
Set image to ESM image and add some more links to run a model page

Whyborn committed:
Accidentally changed preview instead of card image

Whyborn committed:
Specify atmos chemistry as optional for AM3

ccarouge commented:
I am just thinking of a detail: configurations may need some user input before a user can run it. In the Rose/Cylc documentation, we go directly from Get the config to the command to run it. This won't work for AM3. AM3 users need to choose the project to run the suite. So they have to modify the `rose_suite.conf_nci_gadi` file with the info (see [the instructions in README](https://github.com/ACCESS-NRI/access-am3-configs?tab=readme-ov-file#set-your-nci-projects)).

I think we need a sentence in Rose/Cylc docs to warn users to check for each model any specific setup step. And add the steps in the AM3 run docs.

ccarouge committed:
Update run_access-am3.md with setup instructions

Added sections for configuration setup and running the model.

Whyborn committed:
Add JULES bits for AM3

Whyborn commented:
@ccarouge I've also added bits around JULES in AM3. I've kept it very brief, as I don't think that page is the place to get into details about which land model does which bit of science.

ccarouge commented on docs/models/access_models/index.md:
```diff
@@ -6,6 +6,18 @@ Developed in collaboration with international climate modeling institutions, the
+        <div class="card-text-container">
+            <span class="bold" >ACCESS-AM</span>
+            <span>
+                A global atmospheric model that includes atmosphere, land, atmospheric chemistry, aerosols and land biogeochemsitry components.
```
```suggestion
                A global atmospheric model that includes atmosphere, land, atmospheric chemistry, aerosols and land biogeochemistry components.
```

ccarouge commented on docs/models/model_components/bgc_land.md:
```diff
@@ -8,5 +8,6 @@
 The [Carnegie-Ames-Stanford Approach with Carbon-Nitrogen-Phosphorus (CASA-CNP)](https://carbonwaterobservatory.csiro.au/casa.html) is the biogeochemical module implemented in the ACCESS land surface model [CABLE](/models/model_components/land#cable). It models the dynamics of carbon pools and the dependence of carbon uptake due to nitrogen and phosphorous limitations.  
 
 ### Models that use CASA-CNP
+- [ACCESS-AM3](/models/access_models/access-am) (enabled with [CABLE](/models/model_components/land#cable))
```
```suggestion
- [ACCESS-AM3](/models/access_models/access-am) (enabled within [CABLE](/models/model_components/land#cable))
```

ccarouge commented on docs/models/model_components/land.md:
```diff
@@ -11,6 +11,7 @@ Refer to [CABLE documentation](https://cable.readthedocs.io/en/latest/user_guide
 
 ### Models that use CABLE
 
+- [ACCESS-AM3](/models/access_models/access-am#access-am3) (coupled into the [JULES](/models/model_components/land#jules))
```
```suggestion
- [ACCESS-AM3](/models/access_models/access-am#access-am3) (coupled into [JULES](/models/model_components/land#jules))
```

ccarouge commented on docs/models/run_a_model/run_access-am3.md:
```diff
@@ -0,0 +1,69 @@
+{% set github_ssh = "git@github.com:ACCESS-NRI/access-am3-configs.git" %}
+{% set configs_docs = "https://access-am3-configs.access-hive.org.au" %}
+{% set example_branch = "main" %}
+{% set release_notes = "" %}
```
```suggestion
{% set release_notes = "https://forum.access-hive.org.au/t/access-am3-release-information/5446" %}
```

ccarouge reviewed (dismissed):
This looks good to me with a few typo and updates to links.

@heidinett Ready for your review.

heidinett reviewed (commented):
Thanks @Whyborn! I've added a few comments and suggestions.
"Run ACCESS-AM3" page does not appear to be deploying correctly.

Whyborn committed:
Update name of section based on rose/cylc docs bit

Whyborn committed:
Update docs/models/access_models/index.md

Co-authored-by: Claire Carouge <ccarouge@users.noreply.github.com>

Whyborn committed:
Update docs/models/model_components/bgc_land.md

Co-authored-by: Claire Carouge <ccarouge@users.noreply.github.com>

Whyborn committed:
Update docs/models/model_components/land.md

Co-authored-by: Claire Carouge <ccarouge@users.noreply.github.com>

Whyborn commented on docs/models/run_a_model/run_access-am3.md:
```diff
@@ -0,0 +1,69 @@
+
+Before you can run the configuration, you need to specify which projects you want to use for data storage and compute costs. For this, in the configuration you have just retrieved from GitHub, open the `rose-suite.conf_nci_gadi` file and change:
+
+- `root_dir` to the path you want to use as a work directory for running the simulation. A space under /scratch is ideal. The directory will be created by the suite if it does not exist.
```
There's not really any such thing as "your `/scratch` directory`". Perhaps a rewording like:

"A space under `/scratch/<project>` is ideal, where `<project>` is the project associated with the current work."

Whyborn commented on docs/models/run_a_model/run_access-am3.md:
```diff
@@ -0,0 +1,69 @@
+
+If you are unsure whether {{ model }} is the right choice for your experiment, take a look at the overview of [ACCESS Models](/models).
+
+All {{model}} configurations are licensed under the UKMO's [Momentum licence](). {{model}} is delivered to the community through private GitHub repositories. See the [Prerequisites](#prerequisites) section for details.
```
We decided to just not link to anything when we talk about the Momentum license for now.

Whyborn committed:
Minor fixes and rewording end of run a model page

Whyborn committed:
Merge branch '1039-AM3-run-a-model-docs' of github.com:ACCESS-NRI/ACCESS-Hive-Docs into 1039-AM3-run-a-model-docs

Whyborn committed:
Update docs/models/run_a_model/run_access-am3.md

Co-authored-by: Claire Carouge <ccarouge@users.noreply.github.com>

heidinett reviewed (commented):
Looks good @Whyborn! Just suggested one minor change to end of Run ACCESS-AM3.

Whyborn committed:
Specify ACCESS-AM3 at end

ccarouge commented on docs/models/run_a_model/run_access-am3.md:
```diff
@@ -0,0 +1,75 @@
+
+## Inspecting the outputs
+
+The outputs are placed in the directory specified in `STORAGE_PROJECT`, under `cylc-run/<suite-id>`.
```
```suggestion
## Inspecting the outputs

The netCDF outputs are placed in `~/cylc-run/<suite-id>/share/data/History_Data/netCDF`.
The outputs in UMFields format are placed in `~/cylc-run/<suite-id>/share/data/History_Data`.
```

Would that be more useful info?

ccarouge reviewed (dismissed):
I just have a suggestion for the outputs. Everything else looks fine to me.

Whyborn committed:
Update docs/models/run_a_model/run_access-am3.md

Co-authored-by: Claire Carouge <ccarouge@users.noreply.github.com>

Whyborn commented:
@atteggiani Just need your review before merging

Whyborn commented:
@ccarouge Oh we don't? So we just keep any links pointing at the PR preview until beta?

ccarouge commented:
@Whyborn yes.

atteggiani commented on docs/models/run_a_model/index.md:
```diff
@@ -13,6 +13,15 @@ If you are unsure which ACCESS model is the best fit for your application, you c
+        <div class="card-text-container bold">   
+            Run ACCESS-AM
+        </div>
+    </a>
```
```suggestion
    <!-- Run ACCESS-AM3 -->
    <a href="run_access-am3" class="vertical-card aspect-ratio1to1">
        <div class="card-image-container">
            <img class="img-contain with-padding white-background" src="/assets/model-config-logos/model_visualisation/access_esm_model_visualisation.png" alt="ACCESS-AM3">
        </div>
        <div class="card-text-container bold">   
            Run ACCESS-AM3
        </div>
    </a>
```

atteggiani commented on docs/models/access_models/access-am.md:
```diff
@@ -0,0 +1,26 @@
+{% set model = "ACCESS-AM3" %}
+## {{ model }}
+
+{{ model }} is a set of coupled atmosphere-land configurations developed by the ACCESS-NRI, the [Commonwealth Science and Industrial Research Organisation](https://www.csiro.au/en/research/natural-environment?start=0&count=12) (CSIRO) and the [ARC Centre of Excellence for Weather in the 21st Century](https://21centuryweather.org.au/).
```
```suggestion
{{ model }} is a set of coupled atmosphere-land configurations developed by the ACCESS-NRI, the [Commonwealth Science and Industrial Research Organisation (CSIRO)](https://www.csiro.au/en/research/natural-environment?start=0&count=12) and the [ARC Centre of Excellence for Weather in the 21st Century](https://21centuryweather.org.au/).
```

atteggiani commented on docs/models/access_models/access-am.md:
```diff
@@ -0,0 +1,26 @@
+
+{{ model }} is a set of coupled atmosphere-land configurations developed by the ACCESS-NRI, the [Commonwealth Science and Industrial Research Organisation](https://www.csiro.au/en/research/natural-environment?start=0&count=12) (CSIRO) and the [ARC Centre of Excellence for Weather in the 21st Century](https://21centuryweather.org.au/).
+
+ACCESS-NRI has released [{{ model }} configurations](/models/run_a_model/run_access-am3).
```
This link is broken. 
I think, like the other models overview pages, we should link the GitHub repository. In this case, since it is private, we might add minimum information about it and how to get access. Something like:
```suggestion
ACCESS-NRI has released [{{ model }} configurations](https://github.com/ACCESS-NRI/access-am3-configs).

!!! tip
    The configurations GitHub repository is private. To request access to it, follow the related instructions in the [Run {{ model }} page](/models/run_a_model/run_access-am3/#request-access).
```

Note I added the `#request-access` ID in [another comment](https://github.com/ACCESS-NRI/ACCESS-Hive-Docs/pull/1058/files#r2570542777) suggestion.

atteggiani commented on docs/models/access_models/index.md:
```diff
@@ -6,6 +6,18 @@ Developed in collaboration with international climate modeling institutions, the
+        <div class="card-text-container">
+            <span class="bold" >ACCESS-AM</span>
+            <span>
+                A global atmospheric model that includes atmosphere, land, atmospheric chemistry, aerosols and land biogeochemistry components.
```
For ACCESS-rAM3 below we use "atmosphere model".
Should we use "atmospheric model" or "atmosphere model"?
In any case, we should be consistent.

atteggiani replied:
Sounds good! 
Can you please change it when it says "atmosphere model"?

Whyborn replied:
I couldn't find anywhere it said that (incorrectly) in the AM3 docs- the only other instance of "atmosphere model" was where we named AM3 (and in rAM3).

atteggiani replied:
Perfect thank you!

atteggiani commented on mkdocs.yml:
I don't like all the indentation changes. 
Please revert it to the original and only make the specific changes related to this PR.

atteggiani replied:
It seems to be good now.

atteggiani commented on docs/models/access_models/access-am.md:
```diff
@@ -0,0 +1,26 @@
+# ACCESS-AM
```
I would add the Alpha Release banner to these pages.
```
!!! release
    This is an Alpha Release .... link to the "Release" page in /about/releases/
```

atteggiani commented on docs/models/run_a_model/run_access-am3.md:
```diff
@@ -0,0 +1,76 @@
+
+If you are unsure whether {{ model }} is the right choice for your experiment, take a look at the overview of [ACCESS Models](/models).
+
+All {{model}} configurations are licensed under the UKMO's Momentum licence. {{model}} is delivered to the community through private GitHub repositories. See the [Prerequisites](#prerequisites) section for details.
```
```suggestion
All {{model}} configurations are licensed under the UKMO's Momentum licence. {{model}} is delivered to the community through private GitHub repositories. See the [Request access prerequisite](#request-access) for details.
```

atteggiani commented on docs/models/run_a_model/run_access-am3.md:
```diff
@@ -0,0 +1,76 @@
+## Prerequisites
+
+!!! warning
+    If you have not yet read the guide on using the _Rose/Cylc_ tool, please read our [_Rose/Cylc_ documentation](/models/run-a-model/rose-cylc.md) before continuing.
```
```suggestion
!!! warning
    Before continuing, make sure you have read the guide on [running models using _Rose/Cylc_](/models/run-a-model/rose-cylc.md).
```

atteggiani commented on docs/models/run_a_model/run_access-am3.md:
```diff
@@ -0,0 +1,76 @@
+        
+        - already be member of the _ki32_ project
+        {: style="list-style-type: disc"}
+        - have a [MOSRS account](#mosrs-account)
```
```suggestion
        - have a [MOSRS account](/models/run_a_model/rose_cylc/#mosrs-account)
```

atteggiani commented on docs/models/run_a_model/run_access-am3.md:
```diff
@@ -0,0 +1,76 @@
+        - already be member of the _ki32_ project
+        {: style="list-style-type: disc"}
+        - have a [MOSRS account](#mosrs-account)
+        {: style="list-style-type: disc"}
```
```suggestion
```
Same as above. With the CSS addition, this is not needed.

atteggiani commented on docs/models/run_a_model/run_access-am3.md:
```diff
@@ -0,0 +1,76 @@
+        To request membership for the _ki32_mosrs_ subproject, you need to:
+        
+        - already be member of the _ki32_ project
+        {: style="list-style-type: disc"}
```
This works, but in this case I think we always want first-level dot points within an admonition to be rendered this way, so it might be preferable to set a general rule in the CSS file.
I suggest removing this line:
```suggestion
```
 and changing the `access-nri.css`
https://github.com/ACCESS-NRI/ACCESS-Hive-Docs/blob/9e8d910e1a1aef83a78745d4083cce7d1e6df7b6/docs/css/access-nri.css#L460-L466

to
```diff
/* =============================================================== 
  Admonitions 
  (https://squidfunk.github.io/mkdocs-material/reference/admonitions/)
+*/
+
+  .admonition > ul li {
+   list-style-type: disc !important;
+ }

/* General style admonition (In markdown --> `!!! ...`) */
.admonition {
```

atteggiani commented on docs/models/run_a_model/run_access-am3.md:
```diff
@@ -0,0 +1,76 @@
+
+## Get {{model}} configuration
+
+Follow the instructions in the [_Rose/Cylc_ page](/models/run-a-model/rose-cylc.md) using the following specific information in the [Model configurations stored on _GitHub_](/models/run-a-model/rose-cylc/#model-configurations-stored-on-github) section:
```
```suggestion
Follow the instructions for [Model configurations stored on _GitHub_](/models/run_a_model/rose_cylc/#model-configurations-stored-on-github) using the following specific information:
```

atteggiani commented on docs/models/run_a_model/run_access-am3.md:
```diff
@@ -0,0 +1,76 @@
+Before you can run the configuration, you need to specify which projects you want to use for data storage and compute costs. For this, in the configuration you have just retrieved from GitHub, open the `rose-suite.conf_nci_gadi` file and change:
+
+- `root_dir` to the path you want to use as a work directory for running the simulation. A space under `/scratch/<project>` is ideal, where `<project>` is the project associated with the current work. The directory will be created by the suite if it does not exist.
+- `STORAGE_PROJECT` must be the same project as used in the `root_dir` path.
```
This is not necessarily true.
Otherwise, `STORAGE_PROJECT` could be retrieved automatically from `root_dir`, without the need for the user to change/specify two settings.

atteggiani commented on docs/models/run_a_model/run_access-am3.md:
```diff
@@ -0,0 +1,76 @@
+## Inspecting the outputs
+
+The netCDF outputs are placed in `~/cylc-run/<suite-id>/share/data/History_Data/netCDF`.
+The outputs in UMFields format are placed in `~/cylc-run/<suite-id>/share/data/History_Data`.
```
```suggestion
The outputs in [UM Fieldsfiles format](https://code.metoffice.gov.uk/doc/um/latest/papers/umdp_F03.pdf) are placed in `~/cylc-run/<suite-id>/share/data/History_Data`.
```

atteggiani commented on docs/models/run_a_model/run_access-am3.md:
```diff
@@ -0,0 +1,76 @@
+
+## Further Information
+
+See the [ACCESS-AM3 model configuration documentation](https://access-nri.github.io/access-am3-configs-doc/) for more detailed information about the configuration.
```
```suggestion
See [{{ model }} configuration documentation](https://access-nri.github.io/access-am3-configs-doc/) for more detailed information about {{ model }} configuration.
```

atteggiani reviewed (changes requested):
Thank you @Whyborn and @ccarouge for working on this. 

I think it's overall pretty good!
I added some small fixes and suggestions. Please refer to the specific comments for details.

Whyborn commented on docs/models/access_models/access-esm.md:
Not sure, didn't notice this. My editor isn't detecting any differences either. I'll revert this file

atteggiani replied:
This is still showing up.

Whyborn commented on docs/models/index.md:
Yea, neither the web editor nor my editor show any differences here. Something funky is going on. I'll revert these.

atteggiani replied:
This is still showing up.

Whyborn committed:
Apply suggestions from code review

Co-authored-by: Davide Marchegiani <davide.marchegiani@anu.edu.au>

Whyborn committed:
Revert accidental whitespace changes

Whyborn committed:
Merge branch '1039-AM3-run-a-model-docs' of github.com:ACCESS-NRI/ACCESS-Hive-Docs into 1039-AM3-run-a-model-docs

Whyborn committed:
mkdocs additions without unintential whitespace

Whyborn committed:
Add release statement to AM3 pages

Whyborn committed:
Update run_a_model cards with vn number

Whyborn committed:
Apply suggestions from code review

Co-authored-by: Davide Marchegiani <davide.marchegiani@anu.edu.au>

Whyborn committed:
Add cards to top of page

Whyborn committed:
Change to css and associated removals in page

Whyborn committed:
Flesh out release admonition

Whyborn committed:
Update docs/models/run_a_model/run_access-am3.md

Co-authored-by: Davide Marchegiani <davide.marchegiani@anu.edu.au>

Whyborn committed:
Apply suggestions from code review

Co-authored-by: Davide Marchegiani <davide.marchegiani@anu.edu.au>

atteggiani commented on docs/models/run_a_model/run_access-am3.md:
```diff
@@ -0,0 +1,76 @@
+Follow the instructions in the [_Rose/Cylc_ page](/models/run-a-model/rose-cylc.md) using the following specific information in the [Model configurations stored on _GitHub_](/models/run-a-model/rose-cylc/#model-configurations-stored-on-github) section:
+
+- [Repository](/models/run-a-model/rose-cylc/#model-configurations-stored-on-github): {{ github_ssh }}
+- [Branch](/models/run-a-model/rose-cylc/#model-configurations-stored-on-github): {{ example_branch }}
```
Hmm right. Ok for now this is fine then. 
Probably we will need a section on setting up SSH keys for GitHub then (maybe in the getting started page).

atteggiani reviewed (changes requested):
Thank you @Whyborn for addressing my comments. 

The updates look all pretty good! 

Note there are still some files that show here when they should not be within the changed filetree. Also the indentation changes in the mkdocs.yml still seems to be there.

Apart from these minor issues, this PR is ready to be merged.

atteggiani reviewed (dismissed):
Everything looks good now. 

Thank you @Whyborn

atteggiani committed:
Reformatted prerequisites

atteggiani committed:
Fixed broken links

ccarouge committed:
Use 'release-n96e' as example branch instead of "main"

ccarouge committed:
Remove ki32 project requirements

These are not needed anymore now that the SOCRATES files are located under vk83

ccarouge committed:
Forgot the tip on ki32 membership

Removed tip for ki32 membership since that project isn't required anymore

atteggiani committed:
Added readthedocs.yaml

bschroeter commented:
@Whyborn is this now obsolete due to https://github.com/ACCESS-NRI/ACCESS-Hive-Docs/pull/1238 ?

Whyborn commented:
Yea looks like it.
